### PR TITLE
Add group for kustomize

### PR DESCRIPTION
**Pull request by Shell32-Natsu**
This PR is to add staging  group for [kustomize](https://github.com/kubernetes-sigs/kustomize) project.

/cc @monopole

### CODE REVIEW

File: groups/groups.yaml [hunk 1]
```diff
@@ -1506,3 +1506,16 @@ groups:
     members:
       - xuchao@google.com
       - haoweic@google.com
+
+  - email-id: k8s-infra-staging-sigs-kustomize@kubernetes.io
```
Comment (dims) [inline groups/groups.yaml:1510]: Do you need the `-sigs-`?

/hold

**Comment by dims** [comment]
/hold
@Shell32-Natsu this just creates a google group. do you need a GCR/GCS bucket? if so, please see https://github.com/kubernetes/k8s.io/pull/1354

**Comment by Shell32-Natsu** [comment]
I thought that the PRs need to be created separately. Will add them.

**Comment by dims** [comment]
@Shell32-Natsu please look at https://github.com/kubernetes/k8s.io/pull/1347#issuecomment-715468716 there are two people who need to be k8s github org members for this to work. Please work with them to get them org membership (see https://github.com/kubernetes/org/issues/new/choose "Organization Membership Request") or drop them from this PR for now (you can add them later) depending on your urgency.

**Comment by Shell32-Natsu** [comment]
@dims Thanks! When can our staging repo be ready?

**Comment by dims** [comment]
oh! it should be ready already!

**Pull request merged**